stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -454,6 +454,42 @@ def test_potential_qty_list(self):
         )
 
         self.assertEqual(
-            {p1.id: 3.0, p2.id: 3.0, p3.id: 0.0},
+            {p1.id: -1.0, p2.id: 3.0, p3.id: 0.0},
+            {p.id: p.potential_qty for p in products}
+        )
+
+    def test_potential_quantity_service_and_stockable(self):
+        p1 = self.product_model.create({'name': 'Test Product', 'type': 'product'})
+        p2 = self.product_model.create({'name': 'Test PService', 'type': 'service'})
```

Comment:
Nitpicking: Use var names with meaning
```suggestion
        service_product = self.product_model.create({'name': 'Test PService', 'type': 'service'})
```